Random Docs Updates (#683)
tinacms/tinacms.org: content/docs/ui.md

    details: Shows toolbar interface
---

> There are two main pieces of UI that can construct or supplement the content editing interface: the sidebar and the toolbar.

Reviewer: ```suggestion
There are two main UI components that support the content editing interface: the sidebar and the toolbar.
```
just a suggestion to make this sentence a bit simpler
